src/gui/plugins/playback_scrubber/PlaybackScrubber.qml

```diff
@@ -0,0 +1,180 @@
+  columns: 5
+  Layout.minimumWidth: 430
+  Layout.minimumHeight: 200
+  anchors.fill: parent
+  anchors.leftMargin: 10
+  anchors.rightMargin: 10
+  property var isPressed: false
+  property var currentTime: ""
+
+  function updateSliderValue() {
+    if (!playbackScrubber.isPressed)
+    {
+      slider.value = PlaybackScrubber.Progress();
+    }
+  }
+
+  function updateEndTime() {
+    endTime.text = PlaybackScrubber.EndTimeAsString();
+    maxTime.text = qsTr("/ ") + PlaybackScrubber.EndTimeAsString();
+  }
+
+  function updateStartTime() {
+    startTime.text = PlaybackScrubber.StartTimeAsString();
+  }
+
+  function updateCurrentTime() {
+    textField.placeholderText = PlaybackScrubber.CurrentTimeAsString();
+    currentTime = PlaybackScrubber.CurrentTimeAsString();
+  }
+
+  Connections {
+    target: PlaybackScrubber
+    onNewProgress: {
+      updateSliderValue();
+      updateStartTime();
+      updateEndTime();
+      updateCurrentTime();
+    }
+  }
+
+  // Left spacer
+  Item {
+    Layout.columnSpan: 1
+    Layout.rowSpan: 15
+    Layout.fillWidth: true
+  }
+
+  // Top spacer
+  Item {
+    Layout.columnSpan: 3
+    Layout.fillWidth: true
+    height: 12
+  }
+
+  // Right spacer
+  Item {
+    Layout.columnSpan: 1
+    Layout.rowSpan: 15
+    Layout.fillWidth: true
+    width: 30
+  }
+
+  Text {
+    text: "Current time: " + currentTime
+    Layout.columnSpan: 3
+    color: Material.theme == Material.Light ? "black" : "white"
+    font.bold: true
+  }
+
+  Row {
+    Layout.columnSpan: 3
+    spacing: 3
+    Rectangle {
+      height: 40
+      width: 100
+      color: "transparent"
+      Text {
+        id: startTime
+        font.pointSize: 9
+        text: PlaybackScrubber.StartTimeAsString()
+        color: Material.theme == Material.Light ? "black" : "white"
+        anchors.horizontalCenter: parent.horizontalCenter
+        anchors.verticalCenter: parent.verticalCenter
+        
+      }
+    }
+    Rectangle {
+      height: 40
+      width: 200
+      color: "transparent"
+      Slider {
+        anchors.horizontalCenter: parent.horizontalCenter
+        anchors.verticalCenter: parent.verticalCenter
+        id: slider
+        from: 0
+        value: updateSliderValue()
+        to: 1
+        stepSize: 0.001
+        onPressedChanged: {
+          if (!pressed)
+          {
+            PlaybackScrubber.OnDrop(slider.value);
+            playbackScrubber.isPressed = false;
+          }
+          else
+          {
+            playbackScrubber.isPressed = true;
+          }
+        }
+      }
+    }
+    Rectangle {
+      height: 40
+      width: 100
+      color: "transparent"
+      Text {
+        id: endTime
+        font.pointSize: 9
+        text: PlaybackScrubber.EndTimeAsString()
+        color: Material.theme == Material.Light ? "black" : "white"
+        anchors.horizontalCenter: parent.horizontalCenter
+        anchors.verticalCenter: parent.verticalCenter
+      }
+    }
+  }
+
+  TextField {
```

Comment:
I noticed that once I input something in the text field, it stops being updated when I play or seek the simulation. It starts updating again if I delete the text. Is that intentional?

Reply:
Yes, I thought about updating it, but then thought a user might enter a particular time value and keep wanting to go back to it as opposed to having to enter it after every time they play the log back.  Do you think we should just remove the current time preview in the text field?

Reply:
With the new QML updates, your suggestion makes more sense given the empty text box is now the way of determining the current time so I updated it here a6d63aebcbab46eb7e4b99bdd82cd4857f9f6dc0